views/forms/BarChartFields.jsx

```diff
@@ -135,26 +136,31 @@ const BarChartFields = ({ classes, formValues, setFormValues, user }) => {
               <Delete className={classes.icon} />
             </Button>
           </div>
-          {field.targetValues.map((target, tidx) => (
-            <div
-              key={idx + target.site + tidx}
-              className={classes.formLabelRow}
-            >
+          <div className={classes.formLabelRow}>
+            <Typography variant='h6' color='textSecondary'>
+              Targets
+            </Typography>
+          </div>
+          {Object.keys(field.targetValues).map((study, tidx) => (
+            <div key={idx + study + tidx} className={classes.formLabelRow}>
```

Comment:
since studies have unique names I think we could just use `idx + study` here.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/116/commits/3eb84a83fdee7b70f3b433c23cdba25797effe9a